data.js

```diff
@@ -979,5 +979,22 @@ module.exports = [
       'Node.js',
       'Web Development',
     ]
+  },
+  {
+    name: 'Chen A',
+    description: 'Just another blog about Devops/SRE - blog about infrastructure, software-engineering, et al.',
+    url: 'https://devopsian.net',
+    tags: [
```

Comment:
Do you have a twitter account? I think the handle is currently needed { twitter: '@', }.

Or can we hide it when they don't have one @MarkoDenic ?

Reply:
No twitter account :)

Reply:
can't we merge it without a twitter account?